sjsonnet/src-jvm/sjsonnet/Platform.scala

```diff
@@ -24,13 +32,16 @@ object Platform {
     gzipBytes(s.getBytes())
   }
   def xzBytes(b: Array[Byte]): String = {
+    ""
+    /*
     val outputStream: ByteArrayOutputStream = new ByteArrayOutputStream(b.length)
     val xz: XZOutputStream = new XZOutputStream(outputStream, new LZMA2Options())
     xz.write(b)
     xz.close()
     val xzedBase64: String = Base64.getEncoder.encodeToString(outputStream.toByteArray)
     outputStream.close()
     xzedBase64
+    */
```

Comment:
another accidental change?

Reply:
This was to speed up my interation during the benchmark. The file in question spends ~50% of its time in xz - so I quickly just hacked this out.